RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -49,7 +49,7 @@ public override void Fix()
 
         private void RequestLocalCopyVariableName()
         {
-            using( var view = _factory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
+            using( var view = _dialogFactory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
```

Comment:
All the tests pass free of any exceptions - The factory reference is not de-referenced unless the test invokes "Fix()" on the MakeLocalCopyQuickFix.  Once I sort out to use the Moq framework for the factory - I'll pass it to the constructor.